.circleci/config.yml

```diff
@@ -435,6 +435,7 @@ jobs:
 
       - run: |
           # If the current commit is not a tag and if there is no change (compared to main) of anything but docs, then skip this
+          git fetch origin --tags 
```

Comment:
you can add extra guard rails with

```
if ! git fetch origin --tags; then
  touch /tmp/testsuite-failed
fi
```

Reply:
But if the command has a non zero exit code the test suite will fail anyway.